components/features/projects/project-card.server.tsx

```
@@ -0,0 +1,105 @@
+import React from 'react';
+import Image from 'next/image';
+import { ExternalLink as ExternalLinkIcon } from 'lucide-react';
+import type { Project } from '@/types/project';
+import { ExternalLink } from '@/components/ui/externalLink';
+
+// Placeholder for centered top image
+function PlaceholderImageTop() {
+  return (
+    <div className="w-full h-full bg-gray-200 dark:bg-gray-700 flex items-center justify-center text-gray-400 dark:text-gray-500 rounded-md"> {/* Adjusted rounding */}
+      <svg xmlns="http://www.w3.org/2000/svg" className="h-12 w-12" fill="none" viewBox="0 0 24 24" stroke="currentColor"> {/* Adjusted size */}
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)